libs/challenge-registry/user-profile/src/lib/user-profile-overview/user-profile-overview.component.html

```diff
@@ -1,16 +1,14 @@
-<main *ngIf="user" class="main-section-bio-container">
-  <section id="main" class="main-section-bio-main-section">
-    <div class="main-section-bio-container1">
-      <span class="main-section-bio-text">Biography</span>
-      <span class="main-section-bio-text1">{{ user.bio }}</span>
-    </div>
-    <div class="main-section-bio-container2">
-      <h3 class="main-section-bio-text7">ORGANIZATIONS</h3>
-      <challenge-registry-organization-card
+<main *ngIf="user">
+  <div id="bio" class="row col">
+    <h3>Biography</h3>
+    <p>{{ user.bio }}</p>
+  </div>
+  <div id="orgs" class="row">
```

Comment:
I think `col` is needed here instead?
<img width="1104" alt="Screen Shot 2022-09-23 at 12 38 11 PM" src="https://user-images.githubusercontent.com/73901500/192010051-431b682f-07cf-48ab-b950-dfa9f553cc98.png">